config/src/main/resources/goerli.json

```diff
@@ -20,6 +20,12 @@
         "enode://d4f764a48ec2a8ecf883735776fdefe0a3949eb0ca476bd7bc8d0954a9defe8fea15ae5da7d40b5d2d59ce9524a99daedadf6da6283fca492cc80b53689fb3b3@46.4.99.122:32109",
         "enode://d2b720352e8216c9efc470091aa91ddafc53e222b32780f505c817ceef69e01d5b0b0797b69db254c586f493872352f5a022b4d8479a00fc92ec55f9ad46a27e@88.99.70.182:30303"
       ]
+    },
+    "checkpoint": {
```

Comment:
I verified this checkpoint, matches Etherescan

Reply:
This goerli block hash 0x2ae30061bdfc7f6dad5b07361dce436502eb0fde68645de12bae4929be619188 matches Etherscan (on Goerli)